core/src/main/java/org/opensearch/sql/expression/function/OpenSearchFunctions.java

```diff
@@ -48,6 +50,16 @@ private static FunctionResolver match() {
     return new RelevanceFunctionResolver(funcName, STRING);
   }
 
+  private static FunctionResolver matchquery() {
```

Comment:
I don't think this is needed here. If I understand correctly, lucene maps matchquery to match_query when creating a new MatchQuery class in `FilterQueryBuilder.java`. See examples of match_phrase and matchphrase in that file as well. You shouldn't need to classes for match_query in `lucene/relevance` if they have identical implementations.

A good reference here as well would be https://github.com/opensearch-project/sql/pull/604/files#diff-173cbbfbcffec5eb11929c3dd13c124cd81a67f4099c980cb105683e301d62e1 where matchphrase is implemented as well as match_phrase.

Reply:
I was able to remove the duplicate classes in `lucene/relevance` without issue. As for this file, I'm not sure if we should be removing these functions. Many tests begin to fail once these lines are removed, but you could also make the argument that these tests should not exist in the first place. I've commented out the tests for now, but this may be something to discuss further during stand-up

Reply:
Update: I have found that removing these lines causes Integration Tests to fail so I have opted to keep these lines of code.